PractiseMay11/FigureTask/Triangle.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseMay18.FigureTask
+{
+    internal class Triangle : Figure
+    {
+        public double BaseSide { get; set; }
```

Comment:
The setter is excessive for Height and BaseSide since properties are changed in constructor only.